MockFxRatesService.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.jamp2.service.impl;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.epam.jamp2.model.FixerioResponse;
+import com.epam.jamp2.model.UnknownCurrencyException;
+import com.epam.jamp2.rest.FixerioServiceProxy;
+import com.epam.jamp2.service.FxRatesService;
+
+import retrofit2.Response;
+
+public class MockFxRatesService implements FxRatesService {

```

Comment:
Do you use this class anywhere? It's exactly Mockito's purpose to easily build this kind of mocks.